### PR TITLE
If/else docs

### PR DESCRIPTION
**Pull request by psavkar**
## Summary by CodeRabbit

- **New Features**
	- Enhanced documentation for control flow mechanisms, including "If/Else" and advanced operators, with visual aids and clear guidelines.

- **Documentation Improvements**
	- Reorganized pricing documentation for clarity on credit usage and billing model.
	- New metadata files created for control flow concepts to improve navigability and understanding.
	- Streamlined and clarified existing documentation sections for better user experience.

### CODE REVIEW

**Review by andrewjschuang — COMMENTED** [review]
Added some comments and suggestions.

Also, there is an empty file: docs-v2/pages/workflows/control-flow/terminate-workflow.mdx

File: docs-v2/pages/workflows/control-flow/ifelse.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+#### Demo
+
+[VIDEO]
+
+#### Getting Started
+
+Add a trigger and generate a trigger event:
+
+![trigger.gif](https://prod-files-secure.s3.us-west-2.amazonaws.com/6e16aa4c-a31f-4db8-a947-0d80bcdcf984/596d94af-ff09-4a5e-910b-e28db3850890/trigger.gif)
```
[the remark this answers is not in the document]
Comment (psavkar) [reply docs-v2/pages/workflows/control-flow/ifelse.mdx]: This should be fixed.

File: docs-v2/pages/workflows/control-flow/index.mdx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+### Signaling Steps are “Out of Date”
+
+If prior steps in a workflow are modified or retested, Pipedream marks later steps in the execution path as “stale” to signal that the results may be out of date. In the non-linear model, Pipedream only marks steps that are in the confirmed execution path as stale.
+
+- If a change is made to a prior step, then the executed path is cleared.
+    
+    ![Out of date](./images/out_of_date.gif)
+    
+- Steps in the known execution path are immediately marked as stale
+- The state of steps within a conditional blocks are not updated until the path of execution is known. The execution path is known when the start phase is tested and the success branch is identified. At that point, the steps in the known path of execution are marked stale. The test state of steps that are not in the path of execution is preserved.
```
[the remark this answers is not in the document]
Comment (psavkar) [reply docs-v2/pages/workflows/control-flow/index.mdx]: I updated the content.

File: docs-v2/pages/workflows/control-flow/return-http-response.mdx [hunk 3]
```diff
@@ -0,0 +1,145 @@
+import Callout from '@/components/Callout'
+
+# Returning an HTTP Response
```
[the remark this answers is not in the document]
Comment (psavkar) [reply docs-v2/pages/workflows/control-flow/return-http-response.mdx]: Removed. We're moving this to a different section in Phase 2.

**Review by andrewjschuang — APPROVED** [review]
LGTM!

**Pull request merged**